content/authors/eric-ewing/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+# View this page at https://digital.gov/authors/eric-ewing
+# Learn how to edit our pages at https://workflow.digital.gov
+
+# slug — the specific user-id for an author.
+slug: eric-ewing
+display_name: "Eric Ewing"
+first_name: "Eric"
+last_name: "Ewing"
+
+# List your pronoun(s) if you want them displayed alongside your name. If blank, we'll use just your name. Learn more http://mypronouns.org
+pronoun: ""
+
+# Email — If you include an email address, it will be displayed on your profile page
+email: 
+
+# Bio — keep it under 50 words
+bio: "Eric Ewing has over 8 years of experience in data, analytics and artificial intelligence initiatives. Eric has served as a senior advisor to multiple federal Chief Data Officers, providing leadership, guidance, and expertise into enterprise modernization initiatives. With the CoE, Eric consolidates and organizes long and short term strategy for data and AI while leading initiatives for the management of data as a strategic asset. Prior to joining the CoE, Eric was a data scientist and senior consultant leading key analytics programs, AI research and development, and IT supply chain security initiatives with both private and public sector partners."
```

Comment:
Please update the first reference to CoE and the reference to AI to use the full name instead of acronyms.